### ISSUE TITLE
Math.max/ Probably math.min too in LowerBoundChecker

### ISSUE THREAD

**Issue by jsantino** [kelloggm/checker-framework#26]
The Math.max of two values should be the GLB? of the two types
```
void mathmax() {
   @Positive int i = Math.max(-15, 2); // this gives an error but should not
}
```

With Math.min is should be the LUB? of the two

```
void mathmin() {
   @GTENegativeOne int i = Math.min(-1, 2); // this gives an error but should not
}
```

**Comment by kelloggm** [comment]
I don't think this is implemented. I should do it.

**Comment by kelloggm** [comment]
Closed with 7c3f223fb81088cc8be1595d3fc457d9f1e2b26d
